### ISSUE TITLE
Migration guide

### ISSUE THREAD

**Issue by slavaGanzin** [pulsar-edit/pulsar#581]
### Have you checked for existing feature requests?

- [X] Completed

### Summary

Hi

Why pulsar doens't get all config from Atom? Any migration wizard or guide? Almost everybody had some config for Atom and crafting everything from ground up is tedious work. And it kinda kills the whole idea of having compatible editor after sunsetting Atom.

### What benefits does this feature provide?

Healthy inflow of seasoned newcommers

### Any alternatives?

Just use outdated Atom, because it works

**Comment by Spiker985** [comment]
You can simply rename, or copy (preferred) the contents of your `~/.atom` or `%userprofile%/.atom` folder to the pulsar equivalent `.pulsar` within the same locations

However, this is an excellent entry for the FAQ section on the website

We had discussions regarding a migration package previously, but haven't created it

**Comment by Meadowsys** [comment]
yeah, adding the faq entry would be good to have, and would it even be worth creating a migration package? if there are (going to be)  breaking changes then probably, otherwise if it’s just renaming i think it’s unnecessary

**Comment by slavaGanzin** [comment]
If it's so easily done. Why don't just ask user: "do you want to copy your Atom config?" on a startup. 

@Spiker985 thanks for an answer

**Comment by mauricioszabo** [comment]
@slavaGanzin because it's not that simple. We use an outdated Node version, so we don't have a "copy recursive" function; some packages need native modules and these need to be rebuilt (but only after we reload Pulsar); we need to keep track if the uses says "I don't want to migrate" and if the user already migrated; finally, Atom made the decision to _cache_ the initial configurations of the editor, meaning that if we reloaded the editor, we could risk having two "configuration" versions running at the same time, which could corrupt things... and unfortunately, this "cache" flushes to disk when Pulsar _closes_, meaning that copying and closing the editor could indeed corrupt the config.

It is unfortunate, but we inherited a lot of code decisions that we don't fully understand, and that are not so easy to change...

**Comment by savetheclocktower** [comment]
Almost three years later, I'm tempted to close this unless folks think there's still something actionable here. I do agree we should've written better documentation on migration. But if someone came to me and said “how do I migrate from Atom to Pulsar while keeping all my settings and installed packages?” I'd tell them to install `sync-settings`. So if we think it deserves mentioning in the docs still, maybe that can go into an FAQ.
